CsprSdkStandardTestsNet/Test/Steps/GetStateRootHashStepDefinitions.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Casper.Network.SDK;
+using CsprSdkStandardTestsNet.Test.Utils;
+using NUnit.Framework;
+using TechTalk.SpecFlow;
+using static System.Console;
+
+namespace CsprSdkStandardTestsNet.Test.Steps;
+
+/**
+ * Step definitions for get_state_root_hash
+ */
+
+[Binding]
+public class GetStateRootHashStepDefinitions {
+    
+    private static readonly TestProperties TestProperties = new();
+    private readonly Dictionary<string, object> _contextMap = new();
+    private readonly Nctl _nctl = new(TestProperties.DockerName);
+
+    private static NetCasperClient GetCasperService(){
```

Comment:
Formatting space after fu parentheses:
`GetCasperService() {`

Apply to all functions in this class